src/borg/archiver.py

```diff
@@ -1304,6 +1306,8 @@ def build_parser(self, prog=None):
         subparser.add_argument('-e', '--encryption', dest='encryption',
                                choices=('none', 'keyfile', 'repokey'), default='repokey',
                                help='select encryption key mode (default: "%(default)s")')
+        subparser.add_argument('-a', '--append-only', dest='append_only', action='store_true',
+                               help='only allow appending to repository segment files')
```

Comment:
not sure if it is better, but we could also write "create repository in append-only mode", because this is the direct effect of this. what you have is the indirect effect what this will have for later operations.